src/main/java/com/salesforce/emp/connector/BayeuxParameters.java

```diff
@@ -61,7 +61,7 @@ default TimeUnit keepAliveUnit() {
     default Map<String, Object> longPollingOptions() {
         Map<String, Object> options = new HashMap<>();
         options.put("maxNetworkDelay", maxNetworkDelay());
-        options.put("maxBufferSize", maxBufferSize());
+        options.put("maxMessageSize", maxBufferSize());
```

Comment:
Is this right? I think so. We didn't make this change btw. If you inspect the cometd java client **3.1.4**, you'll find references to both maxBufferSize and maxMessageSize properties in ClientTransport and LongPollingTransport classes. In cometd **4.0.4**, only maxMessageSize is referenced.

Reply:
Your analysis sounds reasonable.